.github/workflows/build-and-test.yml

```diff
@@ -17,77 +17,19 @@ permissions: read-all
 jobs:
 
   BuildAndTest:
```

Comment:
All .github/workflows of IPDK will remain as such. The innersource github is on internal server while ipdk.io is on github servers(external)

Reply:
Agree. Will be restoring this file back to original.